### ISSUE TITLE
Which tool did you use for the NMT trainign with BPE-dropout?

### ISSUE THREAD

**Issue by tatHi** [xlhex/dpe#2]
Current Fairseq version does not support any subword regularization like BPE-dropout.
To make same setting reported in your paper, which tool should I use for training of the NMT model?

**Comment by shyyhs** [comment]
I'm using subword-nmt, there's bpe-dropout setting.
https://github.com/rsennrich/subword-nmt

But the data generation is not dynamic and the result is not good. I guess if you want to generate new dropped out data for each epoch, you need to write some code in the train.py.

**Comment by tatHi** [comment]
Thank you very much!
Now I try preparing sampled tokenization for each epoch and dump it as unique files, referring to https://github.com/pytorch/fairseq/issues/1137.
